code/game/gamemodes/game_mode.dm

```diff
@@ -172,7 +172,8 @@
 /datum/game_mode/proc/check_finished() //to be called by ticker
 	if(replacementmode && round_converted == 2)
 		return replacementmode.check_finished()
-	if(SSshuttle.emergency.mode >= SHUTTLE_ENDGAME || station_was_nuked)
+	if((SSshuttle.emergency && SSshuttle.emergency.mode >= SHUTTLE_ENDGAME) \
+		 || station_was_nuked)

```

Comment:
well you could always calculate some subvars

    var/emergencymode = SSshuttle.emergency && SSshuttle.emergency.mode
    if(emergencymode || station_was_nuked)

anyway either way works